spec/controllers/admin/bulk_line_items_controller_spec.rb

```diff
@@ -195,7 +195,7 @@
 
           it "returns an empty JSON response" do
             spree_put :update, params
-            expect(response.body).to eq ' '
+            expect(response.body).to eq ""
```

Comment:
I guess so. I don't know why it would have rendered a string with a space in it before, but it definietly renders an empty string now... :man_shrugging: